test/mbr_update_test.rb

```diff
@@ -134,9 +134,19 @@ def stage1(devices: [], activate: false, generic_mbr: false)
           expect(Yast::Execute).to receive(:locally)
             .with(/parted/, "-s", "/dev/sda", "set", 1, "boot", "on")
           expect(Yast::Execute).to receive(:locally)
-            .with(/parted/, "-s", "/dev/sdb", "set", 1, "boot", "on")
+            .with(/parted/, "-s", "/dev/sdb", "set", 2, "boot", "on")
 
-          subject.run(stage1(activate: true, devices: ["/dev/sda1", "/dev/sdb1"]))
+          subject.run(stage1(activate: true, devices: ["/dev/sda1", "/dev/sdb2"]))
```

Comment:
Any reason for this change from sdb1 to sdb2? Just curious

Reply:
well, for partition it just use any partition which is in stage1, but it does not make any sense to have stage1 in swap partition.

Reply:
so it is just to have reasonable test cases

Reply:
Oh, is explained below

Reply:
yeah, I do not notice it first time. I just use sdb1 in other test case and was surprised why sdb2 was picked and then inspect partitioning which makes it clear :)